src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1628,6 +1628,46 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
             $ref: '#/components/responses/defaultResponse'
+  /license/import:
```

Comment:
I would prefer adding CSV to the endpoint name signifying import file type.
```suggestion
  /license/import-csv:
```

Reply:
done